xds/src/main/java/io/grpc/xds/CdsLoadBalancer.java

```diff
@@ -350,16 +229,33 @@ public void onResourceDoesNotExist(String resourceName) {
 
     @Override
     public void onError(Status error) {
-      logger.log(
-          XdsLogLevel.WARNING,
-          "Received error from xDS client {0}: {1}: {2}",
-          xdsClient,
-          error.getCode(),
-          error.getDescription());
-      if (edsBalancer == null) {
+      logger.log(XdsLogLevel.WARNING, "Received error from xDS client {0}: {1}", error);
```

Comment:
What is {0}?

Reply:
Oops, fixed. Btw, I started to think that including the XdsClient identity in the log is pretty useless..